core/src/test/java/org/apache/jsieve/HeaderTest.java

```diff
@@ -57,6 +57,21 @@ public void testHeaderIsTrue() {
     /**
      * Test for Test 'header'
      */
+    @Test(expected = ThrowTestException.TestException.class)
+    public void testFoldedEncodedHeader() throws Exception {
```

Comment:
@strokyl  sorry I deleted your comment by mistake.

I'll rename this test even if it makes it inconsistent with the other names in the class.

I'll also use ExpectedException

The exception trown is specified in the Sieve script, and it's the common mechanism used in JSieve to test for conditions. I won't change this.